Add often missing parts to "Parts To Self Source"

DerDackel opened the pull request:
Took the first AlieExpress shop link I could find. Needs someone to suggest a better source and/or add a proper affiliate link.

DerDackel committed:
fix: Vz328 Mellow Kit - Kit Contents: Added heat set inserts to Parts To Self-Source section. Needs proper affiliate link

HumbleDeer reviewed (changes requested):
I'd word it differently. These are things you, as a kit builder, have to buy or acquire only because the mellow kit lacks them when the kit should have them. I'd put it under a separate section specifically named and intended for these "Mellow Defects".

We could also dedicate a page to these, to make upkeep easier?

DerDackel committed:
fix: Separate out Mellow defects section with parts we expect to eventually ship with the kit

DerDackel commented:
I like the idea - splitting up things that are to be sourced by the builder vs. stuff that we expect to get into the Mellow kit at one point. I'll use the name Mellow defects for now, but maybe we find a more fitting name for that section.

Do we expect Mellow to eventually ship the filters for the fume extractor too?

HumbleDeer commented on docs/vz330_mellow/kit_contents.md:
```diff
@@ -24,6 +24,7 @@ permalink: /vz330_mellow/kit_contents
 * Electric drill (or manual, if you're so inclined)
 * 3mm metal drill bit
 * M4x1 metric thread tap
+* M3 heat set insert soldering tip
```
I suppose this entry here can be removed, as it's been moved down?

HumbleDeer replied:
I suppose this entry here can be removed, as it's been moved down?

HumbleDeer commented on docs/vz330_mellow/kit_contents.md:
```diff
@@ -248,7 +249,14 @@ The following parts are included in the kit:
 * Raspberry Pi 3/4/Zero ([Official distributors](https://www.raspberrypi.com/resellers/))
-* Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
+
+## Mellow defects
```
Probably ok, but sounds harsh to me?
Maybe "Missing parts"? 
Up to you whether you change it.

HumbleDeer commented on docs/vz330_mellow/kit_contents.md:
```diff
@@ -248,7 +249,14 @@ The following parts are included in the kit:
 * Exhaust filtration / HEPA filter ([Amazon 1](https://amzn.to/3lISPxr) / [AliExpress 1](https://s.click.aliexpress.com/e/_ABbR3U))
 * 2x Front led bars ([AliExpress 1](https://s.click.aliexpress.com/e/_DcRL3RT))
 * Raspberry Pi 3/4/Zero ([Official distributors](https://www.raspberrypi.com/resellers/))
-* Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
```
See next comment about heater fuse

HumbleDeer replied:
See next comment about heater fuse

HumbleDeer commented on docs/vz330_mellow/kit_contents.md:
```diff
@@ -248,7 +249,14 @@ The following parts are included in the kit:
+
+These parts are currently still missing from Mellow's Kit BOM, but should eventually become part of it.
+
+* 1x Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
```
This can also be removed, as it's included in the bed heater itself now. 
The self-sourcing entry is fine to leave in, as it's not in the scope of this pull request or issue. :) 
We can resolve that later; I can't resolve it while you're away because it depends on your PR.

HumbleDeer replied:
This can also be removed, as it's included in the bed heater itself now. 

The self-sourcing entry is fine to leave in, as it's not in the scope of this pull request or issue. :) 
We can resolve that later; I can't resolve it while you're away because it depends on your PR.

HumbleDeer reviewed (changes requested):
/

HumbleDeer commented on docs/vz330_mellow/kit_contents.md:
```diff
@@ -248,7 +249,14 @@ The following parts are included in the kit:
 * Raspberry Pi 3/4/Zero ([Official distributors](https://www.raspberrypi.com/resellers/))
-* Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
+
+## Mellow defects
```
Probably ok, but sounds harsh to me?
Maybe "Missing parts"?
Up to you whether you change it.

DerDackel replied:
Totally agree, "Missing parts" on its own sounds ambiguous though IMO. But I really don't know any really fitting description.

Commit:
Requested changes

This implements the requested changes, as they were so minor as to not be worth bothering the OP.

Commit:
Update kit_contents.md

Minor improvements to aforementioned changes

Commit:
AliExpress link

The link has built in selection for the specific variant/type required (M3x5x4)

Commit:
Remove thermal fuse requirement

Turns out there's no other pages about kit contents, so this is the only one that has this recommendation.
